pkg/objstore/s3/s3.go

```diff
@@ -48,10 +61,24 @@ type Bucket struct {
 	sse    encrypt.ServerSide
 }
 
+// ParseConfig unmarshals a buffer into a Config with default HTTPConfig values
+func ParseConfig(conf []byte) (Config, error) {
+	defaultHTTPConfig := HTTPConfig{IdleConnTimeout: model.Duration(90 * time.Second),
```

Comment:
Let's format it nicely. We need to put newline for each field as the number of elements is large.